inception-project/inception
# #2319 - More user-accessible information about how the recommenders work

**What's in the PR**
- Added confusion matrix as to the recommender sidebar on the annotation page
- Added more helpful information to the evaluation results object so we can access it in the UI
- Make information about when the next evaluation run is due accessible in the UI

**How to test manually**
* Open the recommender sidebar while annotating
* Watch the information presented
* Check out the confusion matrix

**Automatic testing**

**Documentation**
